gecko/glue/include/GeckoMedia.h

```diff
@@ -7,10 +7,19 @@
 #ifndef GeckoMedia_h_
 #define GeckoMedia_h_
 
-struct rust_msg_sender_t;
+struct ThreadObserverVtable
+{
+  void (*on_dispatched_event)(void *data);
+  void (*free)(void *data);
+};
+
+struct ThreadObserverObject {
+  void *data;
```

Comment:
Mozilla C++ style is for struct/class data fields to have a "m" prefix, i.e.: mData.